src/main/core.js

```diff
@@ -171,68 +171,45 @@ function formatRange(text, opts) {
       ...opts,
       rangeStart: 0,
       rangeEnd: Infinity,
-      // track the cursor offset only if it's within our range
+      // Track the cursor offset only if it's within our range
       cursorOffset:
-        opts.cursorOffset >= rangeStart && opts.cursorOffset < rangeEnd
+        opts.cursorOffset > rangeStart && opts.cursorOffset < rangeEnd
```

Comment:
Should not include `=`

Reply:
Can we add test on this?

Reply:
Sure, this is not breaking anything, just no need include `=`.

I'll  add one tomorrow.

Reply:
Done https://github.com/prettier/prettier/pull/9013/commits/055870514fddb9f834e3695b6787501cdfa2f9cf